.github/workflows/ci.yaml

```diff
@@ -62,6 +62,273 @@ jobs:
+            prometheus_url='http://localhost' \
+            trento_server_name='trento-deployment.example.com' \
+            web_admin_password='adminpassword' \
+            enable_api_key='false' \
+            nginx_vhost_listen_port='443' \
+            nginx_ssl_cert_as_base64='true' \
+            nginx_ssl_key_as_base64='true' \
+            nginx_ssl_cert='${{ secrets.SSL_CERT }}' \
+            nginx_ssl_key='${{ secrets.SSL_KEY }}' \
+            install_method='rpm'"
+
+  deploy-test-sp4-rpm:
+    name: Run the playbook on SLES 15 SP4
+    runs-on: ubuntu-latest
+    needs: [build-and-push-container-images]
+    env:
+      TEST_SP4_HOST_IP: ${{ secrets.TEST_SP4_HOST_IP }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install galaxy deps
+        run: ansible-galaxy install -r requirements.yml
+      - name: Run playbook
+        uses: dawidd6/action-ansible-playbook@v2
+        with:
+          playbook: playbook.yml
+          key: ${{ secrets.SSH_MACHINE_KEY }}
+          inventory: |
+            all:
+              vars:
+                ansible_user: ${{ secrets.TEST_HOST_USER }}
+              children:
+                trento-server:
+                  hosts:
+                    server:
+                      ansible_host: ${{ secrets.TEST_SP4_HOST_IP }}
+                postgres-hosts:
+                  hosts:
+                    server:
+                      ansible_host: ${{ secrets.TEST_SP4_HOST_IP }}
+                rabbitmq-hosts:
+                  hosts:
+                    server:
+                      ansible_host: ${{ secrets.TEST_SP4_HOST_IP }}
+          options: |
+            --extra-vars "web_postgres_password='trento' \
+            wanda_postgres_password='wanda' \
+            rabbitmq_password='trento' \
+            prometheus_url='http://localhost' \
+            trento_server_name='trento-deployment.example.com' \
+            web_admin_password='adminpassword' \
+            enable_api_key='false' \
+            nginx_vhost_listen_port='443' \
+            nginx_ssl_cert_as_base64='true' \
+            nginx_ssl_key_as_base64='true' \
+            nginx_ssl_cert='${{ secrets.SSL_CERT }}' \
+            nginx_ssl_key='${{ secrets.SSL_KEY }}' \
+            install_method='rpm'"
+      - name: Test readyness
+        run: curl -k "https://$TEST_SP4_HOST_IP/api/readyz"
+      - name: Run playbook cleanup
+        uses: dawidd6/action-ansible-playbook@v2
+        with:
+          playbook: playbook.cleanup.yml
+          key: ${{ secrets.SSH_MACHINE_KEY }}
+          inventory: |
+            all:
+              vars:
+                ansible_user: ec2-user
+              children:
+                trento-server:
+                  hosts:
+                    server:
+                      ansible_host: ${{ secrets.TEST_SP4_HOST_IP }}
+                      ansible_user: ${{ secrets.TEST_HOST_USER }}
```

Comment:
Do we need this `ansible_user` here?
You don't use it anywhere else

Reply:
Also a leftover, fixed